Makefile

```diff
@@ -18,7 +20,7 @@ check:
 .PHONY: metalint
 metalint:
 	which gometalinter > /dev/null || (go get github.com/alecthomas/gometalinter && gometalinter --install --update)
-	gometalinter --cyclo-over=20 -e "struct field Id should be ID" --enable="gofmt -s" --enable=misspell --fast ./...
```

Comment:
I've added this back, but don't think it is necessary if we move ahead with the refactored struct `PGSession`.